frame/dvm/src/lib.rs

```diff
@@ -419,7 +421,7 @@ impl<T: Config> Pallet<T> {
 
 		Self::deposit_event(Event::Executed(
 			transaction.source,
-			contract_address.unwrap_or_default(),
+			dest.unwrap_or_default(),
```

Comment:
why remove this contract_address? instead by dest?

Reply:
The dest can be either `created_address` when deploying a contract or `to` when making a call.